server/edusync/src/main/java/com/codestates/edusync/model/study/studygroup/service/StudygroupService.java

```diff
@@ -55,19 +74,18 @@ public boolean updateStatus(String email, Long studygroupId) {
 
         if (!findStudygroup.getLeaderMember().getEmail().equals(email)) {
             throw new BusinessLogicException(ExceptionCode.INVALID_PERMISSION);
-        } else {
-            boolean requited = findStudygroup.getIs_requited();
-            if (requited) requited = false;
-            else requited = true;
-            findStudygroup.setIs_requited(requited);
-            studygroupRepository.save(findStudygroup);
         }
-        return findStudygroup.getIs_requited();
+
+        boolean requited = findStudygroup.getIsRecruited();
+        findStudygroup.setIsRecruited(!requited);
+
+        studygroupRepository.save(findStudygroup);
+        return findStudygroup.getIsRecruited();
```

Comment:
깔끔하네요!

Reply:
아 맞다.. 전달드린다는걸 깜빡했네요

Reply:
저거 변수명도 바꿔야하는데...
제안 주시면 반영하겠습니다 ㅋㅋ

Reply:
아니면 그냥 한줄로 처리해도 될 것 같아요.
```suggestion
        findStudygroup.setIsRecruited(!findStudygroup.getIsRecruited());

        studygroupRepository.save(findStudygroup);
        return findStudygroup.getIsRecruited();
```

Reply:
좋습니다~